coffee_cmd/src/main.rs

```diff
@@ -57,8 +57,16 @@ async fn main() -> Result<(), CoffeeError> {
             let remotes = coffee.list().await;
             coffee_term::show_list(remotes)
         }
-        CoffeeCommand::Upgrade { repo } => {
-            match coffee.upgrade(&repo).await {
+        CoffeeCommand::Upgrade { 
+            repo, 
+            verbose, 
+        } => {
+            let spinner = if !verbose {
+                Some(term::spinner("Compiling and installing"))
+            } else {
+                None
+            };
```

Comment:
you need to manage the spinner

Reply:
shall I remove the spinner?

Reply:
https://github.com/coffee-tools/coffee/pull/217/commits/3bc91218e922ff35accd2d5fcee76106481433ab